component/authenticator/src/main/java/org/wso2/carbon/identity/policy/password/PasswordPolicyUtils.java

```diff
@@ -208,270 +190,4 @@ public static String convertWindowsFileTimeToUnixTime(String windowsFileTime) {
         long millisSinceEpoch = (fileTime / HUNDREDS_OF_NANOSECONDS) - WINDOWS_EPOCH_DIFF;
         return String.valueOf(millisSinceEpoch);
     }
-
```

Comment:
Since this is a util method, keep these methods and mark them as deprecated by providing the reason

Reply:
Resolved with https://github.com/wso2-extensions/identity-outbound-auth-passwordPolicy/pull/66/commits/5eb9b2d9f6b7b384d6ee44201bb4aa78d2f8f376